app/components/UI/Rewards/components/Tabs/LevelsTab/UnlockedRewards.tsx

```diff
@@ -114,7 +114,7 @@ const UnlockedRewards: React.FC = () => {
 
   return (
     <Box
-      twClassName="py-4 gap-4"
+      twClassName="py-4 px-4 gap-4"
```

Comment:
nit: could use p-4 here

```suggestion
      twClassName="p-4 gap-4"
```